src/TLuaInterpreter.cpp

```diff
@@ -15449,6 +15450,71 @@ int TLuaInterpreter::getConnectionInfo(lua_State *L)
     return 2;
 }
 
+int TLuaInterpreter::unzip2(lua_State *L)
+{
+    QString zipLocation;
+    if (!lua_isstring(L, 1)) {
+        lua_pushfstring(L, "unzip2: bad argument #1 type (zip location as string expected, got %s!)", luaL_typename(L, 1));
+        return lua_error(L);
+    } else {
+        zipLocation = QString::fromUtf8(lua_tostring(L, 1));
+    }
+
+    QString extractLocation;
+    if (!lua_isstring(L, 2)) {
+        lua_pushfstring(L, "unzip2: bad argument #2 type (extract location as string expected, got %s!)", luaL_typename(L, 2));
+        return lua_error(L);
+    } else {
+        extractLocation = QString::fromUtf8(lua_tostring(L, 2));
+    }
+
+    QTemporaryDir temporaryDir;
+    if (!temporaryDir.isValid()) {
+        lua_pushnil(L);
+        lua_pushstring(L,
+                       "coudln't create temporary directory to extract the zip into");
```

Comment:
`coudln't` ==> `couldn't` {x 2}!

:thought_balloon: Is it better to say "archive" or "zip" here...

Reply:
"zip" is more recognizable by most people

Reply:
Okay.